### PR TITLE
Log environment variable keys at startup

### PR DESCRIPTION
**Pull request by elliesterner**
Log only the keys to environment variables on startup. 
Example:
image

### CODE REVIEW

**Review by ltcarbonell — APPROVED** [review]

File: changelog/18125.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+command/server: Log environment variable keys at startup.
```
Comment (jasonodonnell) [inline changelog/18125.txt:2]:
```suggestion
command/server: Log environment variable keys at startup when log level is set to debug or trace.
```

File: command/server_test.go [hunk 2]
```diff
@@ -203,63 +203,70 @@ func TestServer(t *testing.T) {
...
 			1,
-			"-test-server-config",
+			[]string{"-test-server-config"},
+		},
+		{
+			"environment_variables_logged",
+			testBaseHCL(t, "") + inmemHCL,
+			"Environment Variables",
+			0,
+			[]string{"-test-verify-only", "-log-level", "debug"},
```
Comment (jasonodonnell) [inline command/server_test.go:269]: Can you also add another test for trace level?

**Review by jasonodonnell — CHANGES_REQUESTED** [review]
Small changes but looks good! I did ask a question internally about if putting this in the server configuration info is the right place or if we should simply log it.

File: changelog/18125.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+command/server: Log environment variable keys at startup when log level is set to debug or trace.
```
Comment (jasonodonnell) [inline changelog/18125.txt:2]:
```suggestion
command/server: Environment variable keys are now logged at startup.
```

File: command/server.go [hunk 4]
```diff
@@ -1298,6 +1298,21 @@ func (c *ServerCommand) Run(args []string) int {
 	info := make(map[string]string)
 	info["log level"] = logLevelString
 	infoKeys = append(infoKeys, "log level")
+
+	// returns a slice of env vars formatted as "key=value"
+	envVars := os.Environ()
+	var envVarKeys []string
+	for _, v := range envVars {
+		splitEnvVars := strings.Split(v, "=")
+		envVarKeys = append(envVarKeys, splitEnvVars[0])
```
Comment (jasonodonnell) [inline command/server.go:1307]: This formatting looks wrong, can you run `make fmt` in the root directory and commit the changes (if any)?

**Review by jasonodonnell — APPROVED** [review]
Small nits, but looks good to me!

File: command/server.go (hunk 4 above)
Comment (elliesterner) [reply command/server.go:1307]: that's weird. It looks way more indented in GH than on my local.

**Pull request merged**